packages/app/src/cli/services/app-logs/logs-command/ui/components/hooks/usePollAppLogs.ts

```diff
@@ -88,11 +89,23 @@ export function usePollAppLogs({initialJwt, filters, resubscribeCallback}: UsePo
               return
           }
 
+          const utcDateTime = new Date(log.log_timestamp)
+          const localDateTime = new Date(
+            Date.UTC(
+              utcDateTime.getFullYear(),
+              utcDateTime.getMonth(),
+              utcDateTime.getDate(),
+              utcDateTime.getHours(),
+              utcDateTime.getMinutes(),
+              utcDateTime.getSeconds(),
+            ),
+          )
```

Comment:
I think we could extract this into a method in `utils.ts` given that we want to keep these three callsites consistent